packages/react-ui/src/hooks/useLocale.js

```diff
@@ -0,0 +1,25 @@
+import { createIntl, createIntlCache } from 'react-intl';
+import { messages } from '../localization';
+import { useMemo } from 'react';
+
+const DEFAULT_LANGUAGE = 'en';
+/**
+ * React hook to handle custom locale messages to override default ones
+ * @param {object} localizationMessages
+ */
+export default function useLocale(localizationMessages) {
+  const cache = createIntlCache();
```

Comment:
I'm not sure if the cache-thing should leave here, in the hook. Ideally it should live upper, right?

Reply:
right 👍

Reply:
Done ✅